.gitmodules

```diff
@@ -96,3 +96,13 @@
 	sbom-url = https://github.com/glennrp/libpng.git
 	sbom-description = Portable Network Graphics support, official PNG reference library
 	sbom-hash = 07b8803110da160b158ebfef872627da6c85cbdf
+
+[submodule "freetype/freetype"]
+	path = freetype/freetype
+	url = https://github.com/freetype/freetype.git
+	sbom-version = 2.13.0
+	sbom-cpe = cpe:2.3:a:freetype:freetype:{}:*:*:*:*:*:*:*
+	sbom-supplier = Organization: freetype <https://www.freetype.org>
+	sbom-url = https://github.com/freetype/freetype.git
+	sbom-description = FreeType is a software font engine
+	sbom-hash = 4d8db130ea4342317581bab65fc96365ce806b77
```

Comment:
are you sure this commit sha is correct?

![image](https://github.com/espressif/idf-extra-components/assets/8869469/b2c2628a-c4cc-497d-a9be-fbdd6db38f0a)

Reply:
> are you sure this commit sha is correct?
> 
> ![image](https://user-images.githubusercontent.com/8869469/255054021-b2c2628a-c4cc-497d-a9be-fbdd6db38f0a.png)



> are you sure this commit sha is correct?
> 
> ![image](https://user-images.githubusercontent.com/8869469/255054021-b2c2628a-c4cc-497d-a9be-fbdd6db38f0a.png)

I confirmed that the tag version of `2.13` is indeed `de8b92d`.
I use the master now, I'll test it on this tag, and then submit the modification.